__tests__/cmds/openapi/index.test.ts

```diff
@@ -483,6 +483,40 @@ describe('rdme openapi', () => {
       return mock.done();
     });
 
+    it('should return spec update info for dry run', async () => {
+      prompts.inject(['update', 'spec2']);
+      const registryUUID = getRandomRegistryId();
+
+      const mock = getAPIMock()
+        .get(`/api/v1/version/${version}`)
+        .basicAuth({ user: key })
+        .reply(200, { version })
+        .post('/api/v1/api-registry', body => body.match('form-data; name="spec"'))
+        .reply(201, { registryUUID, spec: { openapi: '3.0.0' } });
+
+      const mockWithHeader = getAPIMockWithVersionHeader(version)
+        .get('/api/v1/api-specification')
+        .basicAuth({ user: key })
+        .reply(200, [
+          { _id: 'spec1', title: 'spec1_title' },
+          { _id: 'spec2', title: 'spec2_title' },
+        ]);
+
+      const spec = './__tests__/__fixtures__/ref-oas/petstore.json';
+
+      await expect(
+        openapi.run({
+          key,
+          version,
+          spec,
+          dryRun: true,
+        })
+      ).resolves.toMatch('dry run! The spec with the id spec2 will be updated');
```

Comment:
you think it's okay to hard code `spec2` here since it's getting hard coded in the `prompts.inject`?

Reply:
Yeah I think that's fine! Alternatively you could create a variable called `specId` and use it like this, but I don't think it really matters honestly:
```suggestion
      ).resolves.toMatch(`dry run! The spec with the id ${specId} will be updated`);
```